apps/api/src/admin/applicant_review_processor.py

```diff
@@ -0,0 +1,69 @@
+from typing import Any
+
+from models.ApplicationData import Decision
+
+
+def include_hacker_app_fields(
+    applicant_record: dict[str, Any], accept_threshold: float, waitlist_threshold: float
```

Comment:
Thought [can handle later]: might be good to define another model for the pre-processed data so we can have stronger type checking rather than relying on `Any`.